lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,380 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
+          # Dependencies (installed in this order)
+          apt_packages: [],
+          brew_packages: [],
+          r_binary_packages: [],
+          r_packages: [],
+          bioc_packages: [],
+          r_github_packages: [],
+          # Build/test options
+          r_build_args: '',
+          r_check_args: '--as-cran',
+          r_check_cran_incoming: false,
+          r_check_revdep: false,
+          # Heavy dependencies
+          pandoc: false,
+          pandoc_version: '1.12.4.2',
```

Comment:
Note to self: update this to `1.13.1` to fix issues with templates being added to the binary.